packages/cinema/cinema.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import utils
+import tmdbsimple as tmdb
+import functools
+import random as ran
+import datetime as dt
+import json
+
+# Package database
+db = utils.db()['db']
+
+# Query
+Query = utils.db()['query']()
+
+
+def load_config(func):
+    @functools.wraps(func)
+    def wrapper_load_config(string, entities):
+        # Init "payload" as dictionary
+        payload = dict()
```

Comment:
As far as I understand, `payload` is configuration right ?
Maybe we could rename it `config` or `configuration` and why do you initialize with `dict()`, you could simply do like this :
```py
config = {}
```

Also as said before, we should avoid using useless comments.